documentation/modules/creating-vddk-image.adoc

```diff
@@ -4,11 +4,13 @@
 
 :_content-type: PROCEDURE
 [id="creating-vddk-image_{context}"]
-= Creating a VDDK image
+= Creating a VDDK image (optional, but strongly recommended)
 
-{the} {project-first} uses the VMware Virtual Disk Development Kit (VDDK) SDK to transfer virtual disks from VMware vSphere.
+{the} {project-first} uses the optional VMware Virtual Disk Development Kit (VDDK) SDK to accelerate transferring virtual disks from VMware vSphere.
```

Comment:
@RichardHoch 
i would suggest something like:
"{the} {project-first} uses the VMware Virtual Disk Development Kit (VDDK) SDK to accelerate transferring virtual disks from VMware vSphere. Therefore, creating a VDDK image, although optional, is highly recommended."